packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -2,6 +2,14 @@
 :host {
   display: block;
 }
+
+:host([disable-border='true']) a[aria-expanded='true'] ~ #info,
+:host([disable-border='']) a[aria-expanded='true'] ~ #info {
+  padding-left: 0;
+  border-left: 0;
+  border-left-color: transparent;
+}
```

Comment:
Is this the most direct approach we could take?

Reply:
@bkjohnson I updated the css to the following to add in the line if the prop is not present or false:
https://github.com/department-of-veterans-affairs/component-library/blob/049b9c4d23b3fef4be19bd0eda58f31e0171709d/packages/web-components/src/components/va-additional-info/va-additional-info.css#L6-L12

I am assuming that is what you were trying to point out via the most recent comment.

If I am incorrect and additional work needs to be done to complete this ticket please let me know what needs to occur and I will gladly adjust. Thanks!